doc/Advanced.md

```diff
@@ -165,6 +168,67 @@ At the moment, the description of the REST api can be found in this document : h
 
 To install and run the QCG locally, and its fellow process tobject2json, please follow these instructions : https://github.com/AliceO2Group/WebUi/tree/dev/QualityControl#run-qcg-locally
 
+## Custom QC object metadata
```

Comment:
I think that two sections are just here because of a merge. Only "Developing QC modules on a machine with FLP suite" should be here. Same in the TOC.

Reply:
oops, indeed

Reply:
done